src/vsr/superblock.zig

```diff
@@ -672,46 +718,44 @@ pub fn SuperBlockType(comptime Storage: type) type {
         }
 
         fn write_staging(superblock: *SuperBlock, context: *Context) void {
```

Comment:
Since `staging` doesn't exist any more, I guess this should be `write_writing`? Unfortunate :D

Reply:
Similarly for `const staging: *SuperBlockSector = superblock.writing;` in several places.

Reply:
Yeah, I left it as `write_staging` because even though we don't have a variable called "staging", we are still (conceptually) writing the staged data.